src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/TdsParserSessionPool.cs

```diff
@@ -89,6 +89,7 @@ internal void Deactivate()
             }
         }
 
+#if NETFRAMEWORK
```

Comment:
NIT: Can you move this netfx-specific section to the end of the class? (It's just a preference, not a required change)